security/bin/e2e.sh

```diff
@@ -50,8 +50,31 @@ if [[ "$HUB" =~ ^gcr\.io ]]; then
   gcloud docker --authorize-only
 fi
 
-# Build and push the Istio-CA docker image
-bin/push-docker.sh -h $HUB -t $TAG
+OUTPUT_DIR=`pwd`/docker
```

Comment:
Could you put these cert-generation commands into a separate file `gen_certs.sh` under security/docker/certs/? Thanks.

Reply:
Done